diagnosticsModule/Private/HelperUtilities.ps1

```diff
@@ -376,6 +376,66 @@ Function GetObjectsFromAD ($domain, $filter, [switch] $GlobalCatalog)
     }
 }
 
+function TryGetDomainNameFromUpn ($Upn)
+{
+    $serviceAccountPartsUpn = $Upn.Split('@')
```

Comment:
Nit picking. The call to this is already splitting the values.

Reply:
Yah, I thought about this.  Ultimately I wanted TryGetDomainNameFromUpn to be one contained unit that was portable.  If we end up using it somewhere else that doesn't need the username I didn't want that to have to split the UPN first before calling this method.  I did think about returning and object with the username and password but I thought that might be overkill.  I will leave as is for now but if there is consensus that it should return an object with the username and password split out I will do that.